olp-cpp-sdk-core/src/http/ios/OLPNetworkIOS.mm

```diff
@@ -50,6 +52,17 @@
     NSString* value = [NSString stringWithUTF8String:header.second.c_str()];
     headers[key] = value;
   }
+
+  // Add OLP user-agent header
+  NSString* custom_user_agent = headers[@(kUserAgentHeader)];
+  NSMutableString* user_agent = [[NSMutableString alloc] init];
+  if (custom_user_agent.length > 0) {
+    [user_agent appendString:custom_user_agent];
+    [user_agent appendString:@"; "];
+  }
+  [user_agent appendString:@(kUserAgentiOS)];
+  headers[@(kUserAgentHeader)] = user_agent;
```

Comment:
You need to check if this is already set and merge

Reply:
I did this already - check lines 57 - 62

Reply:
I see. Thanks